lib/rules/no-mixed-spaces-and-tabs.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
 					 * At least one space followed by a tab
 					 * before non-tab/-space characters begin.
 					 */
-					regex = /^(?=(\t*))\1(?=( +))\2\t/u;
+					regex = /^[ \t]* \t/u;
```

Comment:
Isn't this the same as converting the sequence _lookahead_ + _backreference_ into a regular group?

```suggestion
					regex = /^\t* +\t/u;
```